src/components/Layout.test.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+// import { render, unmountComponentAtNode } from 'react-dom';
+import { render, screen } from '@testing-library/react';
+import Layout from './Layout';
+
+it('Renders a Header component', () => {
+  render(<Layout pageTitle="testTitle" />);
```

Comment:
You're using `act()` in the Footer test--you should add it to these two tests as well.